### PR TITLE
[Added] link token endpoint

### PR DESCRIPTION
**Pull request by anisaoshafi**
:question: What
---

Added link token endpoint

:speech_balloon: Why
---

 
Part of https://github.com/belvo-finance/public_api/issues/1156

### CODE REVIEW

**Review by menecio — CHANGES_REQUESTED** [review]
Please also add it to the documentation of this package

File: belvo/resources.py [hunk 1]
```diff
@@ -135,6 +135,18 @@ def update(
 
         return self.session.put(self.endpoint, id=link, data=data, raise_exception=raise_exception)
 
+    def token(
+        self, link: str, scopes: str, *, raise_exception: bool = False
+    ) -> Union[List[Dict], Dict]:
+        data = {
+            "id": self.session._secret_key_id,
+            "password": self.session._secret_key_password,
```
Comment (AlbertMaristany) [inline belvo/resources.py:143]: Do we send `id` and `password` as body data?

Comment (menecio) [reply belvo/resources.py:143]: You are right @AlbertMaristany , this is how `/api/token/` behaves, not how `/api/link/{id}/token/` should behave

**Review by menecio — CHANGES_REQUESTED** [review]

File: belvo/resources.py (hunk 1 above)
Comment (anisaoshafi) [reply belvo/resources.py]: Removed them.
I tested properly and it works now, the first time with id and password I was a bit blind on that.
I had pending to do some testing, but preferred to open the pr first.
Thanks!

**Comment by anisaoshafi** [comment]
> Please also add it to the documentation of this package

Added.

**Comment by anisaoshafi** [comment]
> Please also add it to the documentation of this package

Added

**Review by AlbertMaristany — APPROVED** [review]
:100:

**Review by menecio — APPROVED** [review]

**Pull request merged**